src/auction/Auction.sol

```diff
@@ -196,10 +202,24 @@ contract Auction is IAuction, VersionedContract, UUPS, Ownable, ReentrancyGuard,
         // If a bid was placed:
         if (_auction.highestBidder != address(0)) {
             // Cache the amount of the highest bid
-            uint256 highestBid = _auction.highestBid;
+            uint256 funds = _auction.highestBid;
+
+            // Check if fee manager is set
+            if (builderFeeManager != address(0)) {
+                // Get the fee amount from the fee manager
+                (address recipent, uint256 fee) = _builderFeeForAmount(funds);
+
+                // If there is a fee to be payed
+                if (fee != 0) {
+                    _handleOutgoingTransfer(recipent, fee);
+                    funds -= fee;
```

Comment:
this will crash if funds < fee due to overflow – do you want an explicit check for that?

Reply:
oh nvm it's a % so that's not possible.